net/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package net provides generic functions that might be useful in
```

Comment:
Maybe this is not the main message the package docs should send. How about:

```
// Package net provides generic network related functions used across Skipper, which
// might be useful also in other contexts than Skipper.
```

?

Reply:
+1